### PR TITLE
Quiz App

### PR DESCRIPTION
**Pull request by Programing-School**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,4 +31,4 @@ What you can make:
 
 ### If you want to contribute a new template you have to follow our [Template Guidelines](https://github.com/mudit023/templates/blob/main/TEMPLATE_GUIDELINES.md).
 
-### Want to contribute to the project? Read the [Contribution Guidelines](https://github.com/mudit023/templates/blob/main/CONTRIBUTING.md) and adhere to our [Code of Conduct](https://github.com/mudit023/templates/blob/main/CODE_OF_CONDUCT.md).
```
Comment (mudit023) [inline README.md:34]: keep your fork repo up to date. There are some changes made in Project README file, don't undo those.

**Review by mudit023 — COMMENTED** [review]
There is just one problem in the README, everything else is fine. Correct that, I'll merge the PR.

File: README.md (hunk 1 above)
Comment (Programing-School) [reply README.md:34]: You want to delete line 34 ??? Because I don't understand

Comment (Programing-School) [reply README.md:34]: @mudit023

Comment (mudit023) [reply README.md:34]: change 
1. "contribution guidelines" to " Contribution Guidelines" 
2.  "code of conduct" to "Code of Conduct"
3. put a full stop sign (period) at the end of line 34

**Review by mudit023 — COMMENTED** [review]
Make the changes

File: README.md (hunk 1 above)
Comment (Programing-School) [reply README.md:34]: > change
> 
> 1. "contribution guidelines" to " Contribution Guidelines"
> 2. "code of conduct" to "Code of Conduct"
> 3. put a full stop sign (period) at the end of line 34

I don't understand, Please Make it

**Comment by Programing-School** [comment]
@mudit023 Success

File: README.md [hunk 2]
```diff
@@ -33,4 +33,15 @@ This project is perfect for people starting out with opensource and knows basics
 <img src="https://github.com/mudit023/templates/blob/main/theme-toggle/Screenshot-Mobile.png" width=400px height=518px align="left">
 <img src="https://github.com/mudit023/templates/blob/main/background-slider/Screenshot-Mobile.png" width=400px>
 
-### Want to contribute to the project? Read the [Contribution Guidelines](https://github.com/mudit023/templates/blob/main/CONTRIBUTING.md), [Template Guidelines](https://github.com/mudit023/templates/blob/main/TEMPLATE_GUIDELINES.md), and adhere to our [Code of Conduct](https://github.com/mudit023/templates/blob/main/CODE_OF_CONDUCT.md).
```
Comment (mudit023) [inline README.md:36]: Again you deleted the change I just made.
See
Screenshot 2022-04-16 012252
**Note:** Always fetch upstream and keep your fork up to date and then only create a PR

**Review by mudit023 — COMMENTED** [review]
Fetch upstream and update your forked repository and then make a new PR

File: README.md (hunk 2 above)
Comment (Programing-School) [reply README.md:36]: Ok

**Comment by Programing-School** [comment]
You can change the readme file with your own code

**Comment by mudit023** [comment]
You are undoing the changes I have made.
Close this PR
Fetch upstream
Create a new PR

**Review by mudit023 — COMMENTED** [review]
There are no screenshots in the template. Add screenshots.

File: README.md [hunk 3]
```diff
@@ -33,4 +33,15 @@ This project is perfect for people starting out with opensource and knows basics
...
+**Note: The first thing you can contribute to is this README.**
```
Comment (mudit023) [inline README.md:36]: Still you are undoing changes i have made in the readme.

**Review by mudit023 — COMMENTED** [review]
Update your forked repository

File: README.md (hunk 3 above)
Comment (Programing-School) [reply README.md:36]: Ok, Please Push these changes to update

Comment (mudit023) [reply README.md:36]: If i will merge this PR the changes I have made in the README will get reverted.

**Comment by Programing-School** [comment]
Please Accept

**Comment by Programing-School** [comment]
I fetch upstream and pulled origin

**Review by mudit023 — APPROVED** [review]
Always make sure that your forked repository is up-to-date

**Comment by Programing-School** [comment]
Ok

**Pull request merged**